src/app/Dashboard/components/DashboardComponent.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { MyStatsComponent, SocietyStatsComponent } from './index';
+import {
+  HeroComponent,
+  NavbarComponent,
+  SidebarComponent,
+  ButtonComponent,
+} from '../../common/components';
+
+const DashboardComponent = () => (
+  <div className='wrapper'>
+    <HeroComponent />
+    <div className='main-content'>
+      <SidebarComponent
+        className='sidebar'
+      />
+      <div className='sub-content'>
+        <NavbarComponent />
+        <div className='user-dashboard'>
+          <h2 className='user-dashboard__name col-sm-12'>Kevin Samoei</h2>
+          <div className='col-sm-12'>
+            <h3 className='user-dashboard__level'>D2</h3>
+          </div>
+          <div className='profile-overview col-sm-12'>
+            <div className='profile-overview__image' />
+            <MyStatsComponent
+              points={1000}
```

Comment:
I believe this is dummy content that will be switched out later no?

Reply:
Yes. This was the goal I set.